### PR TITLE
Use v1 stable for priority class

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/6405

Priority classes graduated to stable in 1.14. So this gets the manifest sorted.

### CODE REVIEW

**Comment by rossf7** [comment]
> This stuff should be validated automatically :/

@corest Heavy +1 :(

I talked with @MarcelMue the current release is big and the priority class is created (using `v1beta1` not `v1alpha1`). I'll merge this as soon as new WIP releases are available.

cc @puja108

**Comment by xh3b4sd** [comment]
@JessicaKCriddle do we have something about automatic api version validation in our board? IMO this is Ludacris' responsibility.

**Comment by JessicaKCriddle** [comment]
Hi @xh3b4sd , would this fall under 'Better release crafting and testing' if so, i've had a look and it doesnt look like it's covered. Please could you check and see what you think and if it's not covered I will add it as a story to this epic. Thank you for the spot. 👀♡

**Comment by xh3b4sd** [comment]
I think it is not covered.

**Comment by JessicaKCriddle** [comment]
ok thanks, i'll add it when I next to refinement for Ludacris. Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ version directory, and  then changes are introduced.
 
 ## [v4.6.0] WIP
 
+### Changed
+
+- Update `giantswarm-critical` priority class manifest to use `v1` stable.
```
Comment (kopiczko) [inline CHANGELOG.md]: I wonder if this is change or fix?

Comment (rossf7) [reply CHANGELOG.md]: Priority classes moved to stable in 1.14 but the v1alpha1 is a bug it should be v1beta1. I'll change to fixed.

Comment (rossf7) [reply CHANGELOG.md]: Done.

Comment (kopiczko) [reply CHANGELOG.md]: Yeah but then isn't this 4.5.1? Sorry for being nitpicky. Maybe that doesn't matter for our internal library.

Comment (rossf7) [reply CHANGELOG.md]: No problem but I'm not sure yet how this will be released. I moved my change to the new WIP version that @MarcelMue added earlier.

Comment (kopiczko) [reply CHANGELOG.md]: Sure this is WIP. Ignore me. Thanks.

**Comment by rossf7** [comment]
Tested on `gauss`.

**Pull request merged**